generated/nidcpower/session.py

```diff
@@ -546,7 +596,8 @@ class _SessionBase(object):
     This attribute is not available in a session involving multiple channels.
     '''
     measure_record_length_is_finite = attributes.AttributeViBoolean(1150064)
-    '''
+    '''Data Type: bool
```

Comment:
Very minor and personal preference, I think "Type" suffices.

Reply:
Fixed